am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/DefaultRoleSetupDataFactory.java

```diff
@@ -30,9 +30,9 @@ private DefaultRoleSetupDataFactory() {
             new Pair<>(permissions, SecurityClassification.PUBLIC);
 
         Map<JsonPointer, Map.Entry<Set<Permission>, SecurityClassification>> attributePermission =
-            new ConcurrentHashMap<>();
-
-        attributePermission.put(attribute, pair);
+            ImmutableMap.<JsonPointer, Map.Entry<Set<Permission>, SecurityClassification>>builder()
+                .put(attribute, pair)
+                .build();
```

Comment:
ImmutableMap.of() would be better and less here

Reply:
👍